src/renderer/components/+custom-resources/certmanager.k8s.io/index.ts

```diff
@@ -1,4 +0,0 @@
-export * from "./certificates"
```

Comment:
Removed everything related to `certmanager` because it's using old apiversion and this actually should be an extension.